mackerel-plugin-apache2/flags.go

```diff
@@ -42,7 +44,18 @@ var cliStatusPage = cli.StringFlag{
 
 var cliTempFile = cli.StringFlag{
 	Name:   "tempfile, t",
-	Value:  "/tmp/mackerel-plugin-apache2",
 	Usage:  "Set temporary file path.",
 	EnvVar: "ENVVAR_TEMPFILE",
 }
+
+var cliMetricKerPrefix = cli.StringFlag{
+	Name:  "metric-key-prefix, m",
+	Value: "apache2",
+	Usage: "Set metric key prefix.",
+}
+
+var cliLabelPrefix = cli.StringFlag{
+	Name:  "label-prefix, l",
```

Comment:
How about using `metric-label-prefix` instead to make it compatible with other plugins?
https://github.com/mackerelio/mackerel-agent-plugins/search?utf8=%E2%9C%93&q=metric-label-prefix